src/lib/tokenizer.js

```diff
@@ -0,0 +1,88 @@
+class Tokenizer {
+  static theTokenizer;
+
+  constructor(content, literalsList) {
+    this.program = content;
+    this.literals = literalsList;
+    this.tokens = [];
+    this.currentToken = 0;
+
+    this.tokenize();
+  }
+
+  tokenize() {
+    let tokenizedProgram = this.program;
+    tokenizedProgram = tokenizedProgram.replace(/\n/g, '');
+    console.log(this.program);
+
+    this.literals.forEach(
+        s => {
+          tokenizedProgram = tokenizedProgram.replace(s, `_${s}_`);
```

Comment:
This currently only replaces the first instance of string literals, is `literals` supposed to be regex? I'm assuming `literals` should be `string[]`

Reply:
Fixed in aa1c171